go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cycloidio/cycloid-cli
 
-go 1.17
+go 1.22
```

Comment:
This could be better to upgrade go in a separate PR, does your code requires that upgrade to work ?

I would do the update on a separate pr to easily rollback the changes in case of compatibility issue.

Reply:
Do it, test it, good luck for the upgrade :)